src/app/hooks/usePasswordForm.tsx

```diff
@@ -1,7 +1,7 @@
 import { useRouter } from "next/router";
 import {useForm} from 'react-hook-form';
 import { yupResolver } from "@hookform/resolvers/yup";
-import {TPasswordForm } from '../types';
+import {TPasswordForm, TUserData } from '../types';
```

Comment:
I must have removed the TUserData types because it was not in used. Remember any unused elements, types, variables, parameters, functions must all be removed.